pkg/ifuzz/arm64_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package ifuzz
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/google/syzkaller/pkg/ifuzz/arm64"
+	_ "github.com/google/syzkaller/pkg/ifuzz/arm64/generated"
```

Comment:
ifuzz.go already imports this, is this necessary?

Reply:
Actually no :)